hack/get-cd-registry.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+echo "eu.gcr.io/sap-se-gcr-k8s-private/cnudie/gardener/development"
```

Comment:
Does a reference to a private repository make sense in an open source project?

Reply:
I have replaced the URL by a hint to insert the location for the component descriptor.